From 87af0fdec4f998efa11d90622dbe317f46d165be Mon Sep 17 00:00:00 2001
From: Kendall Willis <k-willis@ti.com>
Date: Mon, 1 Dec 2025 15:23:37 -0600
Subject: [PATCH 2/2] drm/panel: simple: Halve the pixel clock rate of
 Raspberry Pi 7" DSI panel

The pixel clock of the Raspberry Pi 7" DSI panel is halved for
increased power savings.

Signed-off-by: Kendall Willis <k-willis@ti.com>
---
 drivers/gpu/drm/panel/panel-simple.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/panel/panel-simple.c b/drivers/gpu/drm/panel/panel-simple.c
index 29838b6de12b..a5dfd677f583 100644
--- a/drivers/gpu/drm/panel/panel-simple.c
+++ b/drivers/gpu/drm/panel/panel-simple.c
@@ -3855,7 +3855,7 @@ static const struct panel_desc rocktech_rk043fn48h = {
 };
 
 static const struct drm_display_mode raspberrypi_7inch_mode = {
-	.clock = 28569600 / 1000,
+	.clock = 28569600 / 2000,
 	.hdisplay = 800,
 	.hsync_start = 800 + 48,
 	.hsync_end = 800 + 48 + 32,
-- 
2.34.1

